src/components/CodeViewer.jsx

```
@@ -1,27 +1,83 @@
+import Editor from "@monaco-editor/react";
+
+const CodeViewer = ({ code, onChange }) => {
+  const editorRef = useRef(null);
+
+  useEffect(() => {
+    if (editorRef.current) {
+      editorRef.current.setValue(code);
+    }
+  }, [code]);
+
+  function handleEditorDidMount(editor, monaco) {
```

ESLint

[error] 'monaco' is defined but never used.

(no-unused-vars)